### PR TITLE
Update Helm 3 doc to improve usability

### PR DESCRIPTION
**Pull request by hickeyma**
I was using the [Helm3 support](https://github.com/istio/istio/blob/master/manifests/charts/README-helm3.md) doc to install Istio into a cluster with Helm. The doc is good but I found a few things when doing it. I have therefore added some suggestions for improvement which could help someone new to Istio, to better navigate the process.

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[x] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: manifests/charts/README-helm3.md [hunk 1]
```diff
@@ -1,57 +1,64 @@
...
+helm install istio-base -n istio-system manifests/charts/base
 ```
 
 - 'istio-control/istio-discovery' installs a revision of istiod.  You can install it multiple times, with different revisions.
-TODO: remove the need to pass -n istio-system
 
-```shell script
- helm3 install -n istio-system istio-17 manifests/charts/istio-control/istio-discovery
+```console
+ helm install -n istio-system istio-17 manifests/charts/istio-control/istio-discovery -f manifests/charts/global.yaml
```
Comment (howardjohn) [inline manifests/charts/README-helm3.md:25]: I thought we removed the requirement for this. @jacob-delgado is it still required, and if not can it be removed?

Comment (morvencao) [reply manifests/charts/README-helm3.md:25]: It should be removed after https://github.com/istio/istio/pull/25186

File: manifests/charts/README-helm3.md [hunk 2]
```diff
@@ -1,57 +1,64 @@
-# Helm3 support
+# Helm v3 support
 
 ## Install
 
-The manifests/ templates support both helm2 and helm3. Please do not introduce helm3-specific changes, many
-users are still using helm2 and the operator is currently using the helm2 code to generate.
+The Helm charts are support both by Helm v2 and Helm v3. Please do not introduce Helm v3 specific changes as many
```
Comment (shamsher31) [inline manifests/charts/README-helm3.md:5]:
```suggestion
The Helm charts are supported both by Helm v2 and Helm v3. Please do not introduce Helm v3 specific changes as many
```

File: manifests/charts/README-helm3.md [hunk 3]
```diff
@@ -1,57 +1,64 @@
...
+```console
+kubectl create namespace istio-system
 ```
 
-We have few charts:
+The charts are as follows:
 
-- 'base' creates cluster-wide CRDs, cluster bindings, cluster resources and the istio-system namespace.
-  It is possible to customize the namespace, but not recommended.
+- 'base' creates cluster-wide CRDs, cluster bindings and cluster resources. It is possible to change the namespace from `istio-system` but it is not recommended.
```
Comment (shamsher31) [inline manifests/charts/README-helm3.md:16]:
```suggestion
- `base` creates cluster-wide CRDs, cluster bindings, and cluster resources. It is possible to change the namespace from the `istio-system` but it is not recommended.
```

File: manifests/charts/README-helm3.md [hunk 4]
```diff
@@ -1,57 +1,64 @@
...
+ helm install -n istio-system istio-canary manifests/charts/istio-control/istio-discovery \
     -f manifests/charts/global.yaml  --set revision=canary --set clusterResources=false
 
- helm3 install -n istio-system istio-mytest manifests/charts/istio-control/istio-discovery \
+ helm install -n istio-system istio-mytest manifests/charts/istio-control/istio-discovery \
     -f manifests/charts/global.yaml  --set revision=mytest --set clusterResources=false
 ```
 
-- 'ingress' to install a Gateway
+- 'gateways' install a load balancer with `ingress` and `egress`. You can install it multiple times with different revisions but they must be installed in separate namespaces.
```
Comment (shamsher31) [inline manifests/charts/README-helm3.md:34]:
```suggestion
- `gateways` install a load balancer with `ingress` and `egress`. You can install it multiple times with different revisions but they must be installed in separate namespaces.
```

File: manifests/charts/README-helm3.md [hunk 5]
```diff
@@ -1,57 +1,64 @@
...
-  It is possible to customize the namespace, but not recommended.
+- 'base' creates cluster-wide CRDs, cluster bindings and cluster resources. It is possible to change the namespace from `istio-system` but it is not recommended.
 
-```shell script
- helm3 install istio-base -n istio-system manifests/charts/base
+```console
+helm install istio-base -n istio-system manifests/charts/base
 ```
 
 - 'istio-control/istio-discovery' installs a revision of istiod.  You can install it multiple times, with different revisions.
```
Comment (shamsher31) [inline manifests/charts/README-helm3.md:22]:
```suggestion
- `istio-control/istio-discovery` installs a revision of `istiod`.  You can install it multiple times, with different revisions.
```

File: manifests/charts/README-helm3.md (hunk 1 above)
Comment (hickeyma) [reply manifests/charts/README-helm3.md:25]: That is a typo by me, apologies. It is working without the global values.

**Comment by hickeyma** [comment]
I will do an update on the CLA and reviews asap.

File: manifests/charts/README-helm3.md (hunk 1 above)
Comment (hickeyma) [reply manifests/charts/README-helm3.md]: Removed.

File: manifests/charts/README-helm3.md (hunk 2 above)
Comment (hickeyma) [reply manifests/charts/README-helm3.md]: Updated

File: manifests/charts/README-helm3.md (hunk 3 above)
Comment (hickeyma) [reply manifests/charts/README-helm3.md]: Done

File: manifests/charts/README-helm3.md (hunk 4 above)
Comment (hickeyma) [reply manifests/charts/README-helm3.md]: Done

File: manifests/charts/README-helm3.md (hunk 5 above)
Comment (hickeyma) [reply manifests/charts/README-helm3.md]: Done

**Comment by hickeyma** [comment]
Thanks for the reviews @howardjohn @morvencao @shamsher31. Updated and ready for review again.

**Review by sdake — APPROVED** [review]

**Pull request merged**